west.yml

```diff
@@ -248,6 +250,12 @@ manifest:
       path: modules/hal/ti
       groups:
         - hal
+    - name: ch32v003fun
+      remote: cnlohr
+      revision: pull/364/head
```

Comment:
This is strange - do you want to pin to a specific hash or something?  Just FYI - we will likely do a proper release after the 002, 004, 005, and 006 release.

Or is this really how it's supposed to go?

Reply:
It's how we do module updates: this is just a temporary thing to have the Zephyr CI pick up the change and run on the live module PR, our automation detects this and tags the Zephyr PR as DNM automatically, but in the meantime we can validate that the CI passes with the change. Then once the module PR is merged this gets replaced with the actual SHA.

Full process: https://docs.zephyrproject.org/latest/develop/modules.html#submitting-changes-to-modules

Reply:
And yeah since it's been merged: @kholia you can use `2d464950429504417a2cb33bc5313739500e518e` now, then I guess you may want to also cherry pick it into https://github.com/zephyrproject-rtos/zephyr/pull/76540 and I'll ask the TSC to create the repo next week.

Reply:
Sorry, that I can't do much else to review this PR, as I am unfamiliar with most of the inner workings of Zephyr.

Reply:
That would be a bit of a nightmare :-) no it's just so we can run the full upstream CI and verify it passess before the module one is merged, which is exactly what we did (apart from the compliance error but that's just the repo name)